core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4294,13 +4294,13 @@ protected function parentView()
 					$this->import($strClass);
 					$options_callback = $this->$strClass->$strMethod($this);
 
-					$_v = $options_callback[$_v];
+					$_v = $options_callback[$_v] ?? '---';
```

Comment:
```suggestion
					$_v = $options_callback[$_v] ?? '-';
```

The fallback value seems to be `'-'` in most conditions above, so we should keep this consistend I tink?

Reply:
okay, I solved that although I thought the blank option is `---` in some cases